bundles/org.openhab.binding.pentair/README.md

```diff
@@ -72,133 +87,200 @@ The following table shows the available configuration parameters for each thing.
+| "                                 | temperature | Number:Temperature | R | Current water temperature.  Note, the temperature is only valid while in either pool or spa mode. |
+| "                                 | heatmode  | String | R  | Heat mode configured.  Values: NONE, HEATER, SOLARPREFERRED, SOLAR |
+| schedule[1-9]                     | schedule  | String | RW | Summary string of schedule.  |
+| "                                 | type      | String | RW | Type of schedule.  Note, to actually write the program to the controller, this channel must be written to with the same value 2 times within 5s. Values: NONE, NORMAL, EGGTIMER, ONCE ONLY |
+| "                                 | start     | Number | RW | Time of day to start schedule expressed in minutes.  |
+| "                                 | end       | Number | RW | Time of day to end schedule expressed in minutes. In the case of EGG TIMER, this shoud be the duration. |
+| "                                 | circuit   | Number | RW | Circuit/Feature the schedule will control. |
+| "                                 | days      | String | RW | The days the schedule will run.  S=Sunday, M=Monday, T=Tuesday, W=Wednesday, R=Thursday, F=Friday, Y=Saturday |
+| status                            | lightmode | String | RW | Light mode. Values: OFF, ON, COLORSYNC, COLORSWIM, COLORSET, PARTY, ROMANCE, CARIBBEAN, AMERICAN, SUNSET, ROYAL, BLUE, GREEN, RED, WHITE, MAGENTA |
+| "                                 | solartemperature | Number:Temperature | R | Solar temperature sensor reading. |
+| "                                 | airtemperature   | Number:Temperature | R | Air temperature sensor reading. |
+| "                                 | uom              | String             | R | Unit of measure. Values: CELCIUS, FARENHEIT. |
+| "                                 | servicemode      | Switch             | R | Indicates whether controller is in service mode. |
+| "                                 | solaron          | Switch             | R | Indicates whether solar heat is on. |
+| "                                 | heateron         | Switch             | R | Indicates whether heater is on. |
+
+#### Working with schedules
+
+This binding allows both reading and writing of schedules and supports up to 9 schedules. Programming of a schedule can be accomplished either by using the discrete channels linked to items (i.e. type, start, end, circuit, days) or you can concatenate those and use the `schedule` channel saved as a comma delimited string.  To prevent erroneous writes to the schedules though, one must write to the `type` channel the same value twice within 5 sec.
+
+### Thing: IntelliChlor
+
+Represents an Intellichlor module connected in your system.  Currently, the values here are readonly.
+
+| Channel              | Type       |     | Description |
+| :------------------: | :----:     | :-: | :---------- |
+| saltoutput           | Number     | R   | Current salt output %. |
```

Comment:
Percent values could be `Number:Dimensionless`. If you send them with `QuantityType.valueOf(value, Units.PERCENT)`, they are automatically displayed as percent values in the UI.